CONTRIBUTING.md

```diff
@@ -28,6 +28,11 @@ First of all, thank you for contributing to Meilisearch! The goal of this docume
 
 ### Setup <!-- omit in toc -->
 
+You may opt to setup your local environment natively or using `docker`, check out the [`docker-compose.yml`](/docker-compose.yml) configuration.
+
+Either natively or with docker you will run the same commands below: 
+
```

Comment:
I know we don't all agree on this but I think we should at least write one command to launch it. I vote for the one to launch the tests. If we don't want to add it to the `CONTRIBUTING` file maybe we can add it as a comment in the `docker-compose.yml` file?

Reply:
I think we should add one docker command inside the contributing guide. The one that launches all the tests. I would always use it to copy paste.